src/main/java/view/impl/UsersMenu.java

```diff
@@ -0,0 +1,71 @@
+package view.impl;
+
+import dao.UserDao;
+import model.User;
+import service.UserService;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Scanner;
+
+
+public class UsersMenu {
+    public void runMenu() {
+
+        int input = showMainMenu();
+
+        if (input == 0) {
+            return;
+        } else if (input == 1) {
+            blockUsersMenu(true);
+        } else if (input == 2) {
+            blockUsersMenu(false);
+        } else if (input == 3) {
+            //TODO Create massages
+            System.out.println("Select user to sent the message");
+        }
+    }
+
+    public int showMainMenu() {
+        System.out.println("Select menu item:");
+        System.out.println("1. Block user");
+        System.out.println("2. Unblock user");
+        System.out.println("3. Private message");
+        System.out.println("0. To exit");
+
+        Scanner scanner = new Scanner(System.in);
+        return scanner.nextInt();
+    }
+
+    public void blockUsersMenu(boolean block) {
+        List<User> users;
+        if (block) {
+            System.out.println("Select user to block");
+        } else {
+            System.out.println("Select user to unblock");
+        }
+        users = UserDao.getBlockedUsersList(!block);
+        Map<Integer, Integer> usersId = new HashMap<>();
```

Comment:
use java 11 syntax

Reply:
I can not find what coul I chande in this string... And IDE does not suggest anything....
`Map<Integer, Integer> usersId = new HashMap<>();
`

Reply:
var usersId = new HashMap<Integer, Integer>();

Reply:
fixed